components/Notifications/Notification.tsx

```diff
@@ -78,12 +78,7 @@ const IconWrapper = styled.div`
   height: 32px;
 `;
 
-const SuccessIcon = styled(Success)`
-  path {
-    fill: var(--white);
-    stroke: var(--green);
-  }
-`;
+const SuccessIcon = styled(Success)``;
```

Comment:
I think these are old styles. the icon looks right as is without them, and having them cuts off the edges of the circle